recipes/gfortran_osx-64/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set gfortran_version = [7, 3, 0] %}
+{% set libgfortran_version = [4, 0, 0] %}
+{% set max_libgfortran_version = "5.0.0.a0" %}
+
+package:
+  name: gfortran_osx-64
+  version: {{ gfortran_version|join('.') }}
+
+build:
+  number: 0
+  skip: True  # [not osx]
+  run_exports:
+    strong:
+      - libgfortran >={{ libgfortran_version|join('.') }},<{{ max_libgfortran_version }}
+
+requirements:
+  run:
+    - gfortran_impl_osx-64 =={{ gfortran_version|join('.') }}
+    - libgfortran >={{ libgfortran_version|join('.') }},<{{ max_libgfortran_version }}
+
+test:
+  files:
+    - hello.f90
+    - maths.f90
+  commands:
+    - gfortran --help
+
+about:
+  home: http://gcc.gnu.org/
+  license: GPL 3 (with GCC Runtime Library Exception 3.1)
+  license_family: GPL
+  summary: Fortran libraries from the GNU Compiler Collection
+
+extra:
+  recipe-maintainers:
+    - jakirkham
+    - msarahan
+    - pelson
+    - beckermr
```

Comment:
@msarahan @pelson @jakirkham - Are you all OK with being maintainers here?

Reply:
Bump on this friends!

Reply:
Shall we remove the other maintainers and merge @isuruf @scopatz?

Core will have access no matter what and other ppl can be added as needed.